projects/vdk-plugins/vdk-jupyter/setup.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import pathlib
+
+import setuptools
+
+"""
+Builds a package with the help of setuptools in order for this package to be imported in other projects
+"""
+
+__version__ = "0.1.0"
+
+setuptools.setup(
+    name="vdk-jupyter",
+    version=__version__,
+    url="https://github.com/vmware/versatile-data-kit",
+    description="Plugin for runnig Jupyter notebooks as vdk steps",
+    long_description=pathlib.Path("README.md").read_text(),
+    long_description_content_type="text/markdown",
+    install_requires=["vdk-core"],
+    package_dir={"": "src"},
+    packages=setuptools.find_namespace_packages(where="src"),
+    # This is the only vdk plugin specifc part
+    # Define entry point called "vdk.plugin.run" with name of plugin and module to act as entry point.
+    entry_points={"vdk.plugin.run": ["jupyter = vdk.plugin.jupyter.jupyter_plugin"]},
+    classifiers=[
+        "Development Status :: 4 - Beta",
```

Comment:
The Development Status should be 2 Pre-Alpha (as defined here https://martin-thoma.com/software-development-stages/#2-pre-alpha) that seems the best fit currently.